packages/module/sass/react-catalog-view-extension/_catalog-tile.scss

```diff
@@ -27,7 +27,7 @@
   }
 
   .catalog-tile-pf-subtitle {
-    color: var(--pf-v6-global--Color--200);
+    color: var(--pf-t--global--border--color--brand--default);
```

Comment:
Looks like this is just a subtitle/description block with secondary/grey text, so you can use `--pf-t--global--text--color--subtle`